src/test/java/cz/centrum/haffner/SimpleTrainingDavid/SimpleTrainingDavidApplicationTests.java

```diff
@@ -1,9 +1,10 @@
 package cz.centrum.haffner.SimpleTrainingDavid;
 
 import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.test.context.SpringBootTest;
 
-
+@SpringBootApplication
```

Comment:
@SpringBootConfiguration bude stačit. (je součástí @SpringBootApplication, ale přímo ta, není potřeba)

Reply:
předěláno ...